containermessage/models.go

```diff
@@ -151,6 +151,8 @@ type Service struct {
 	Ports            []docker.PortBinding `json:"ports,omitempty"`
 	EphemeralPorts   []Port               `json:"ephemeral_ports,omitempty"`
 	SpecificPorts    []docker.PortBinding `json:"specific_ports,omitempty"` // obselete. for backward compatibility only, new way should use ports instead.
+	MaxMemoryKb      int64                `json:"max_memory_kb,omitempty"`
```

Comment:
Please update one of the services created by e2edev to specify a memory and cpu limit. And then add code to verify that docker inspect of that service (when it's running) shows the cpu and memory limits you specified.

Reply:
Updated PR with additional memory&cpu check